### PR TITLE
Fix error on repeat calls to DEVICE_GetTemperature

### PR DESCRIPTION
**Pull request by aromanielloNTIA**
Fix an error which occurs when calling `DEVICE_GetTemperature` multiple times. This happened due to an incorrectly implemented check for the length of the frequency reference user setting string.

Bumps version to 1.3.2, including this fix

### CODE REVIEW

File: src/rsa_api/rsa_api.py [hunk 1]
```diff
@@ -620,10 +623,7 @@ def CONFIG_SetFreqRefUserSetting(self, i_usstr: Union[str, None] = None) -> None
             self.err_check(self.rsa.CONFIG_SetFreqRefUserSetting(None))
         else:
             RSA.check_string(i_usstr)
-            if (
-                i_usstr == "Invalid User Setting"
-                or len(i_usstr) != _FREQ_REF_USER_SETTING_STRLEN
-            ):
+            if i_usstr == "Invalid User Setting":
```
Comment (jhazentia) [inline src/rsa_api/rsa_api.py:626]: Would there still be a need to check that user string has length less than _FREQ_REF_USER_SETTING_STRLEN?

Comment (aromanielloNTIA) [reply src/rsa_api/rsa_api.py:626]: Good point. Added this check in [ef39a16](https://github.com/NTIA/tekrsa-api-wrap/pull/16/commits/ef39a1602844af50c8755d6193e2e881cb41f20b), with its own informative error message

**Comment by aromanielloNTIA** [comment]
> 



> Probably unrelated to this change, but the test_IQBLK_GetIQData test seems to be hanging when tested on sensor.

I've seen this before. I'm not sure why it happens, because you can create a standalone script outside of the unit test and run the same code, and it all works as expected. So this is an issue with the test, not the API wrapper itself. The same problem occurs for testing the spectrum acquisition. I've removed these tests temporarily (in [7c64925](https://github.com/NTIA/tekrsa-api-wrap/pull/16/commits/7c64925db7fb006b238ccbd11d03bf51e9e82155)) and will open issues to address these separately

**Review by jhazentia — APPROVED** [review]
Looks good!

**Pull request merged**